### PR TITLE
mimirtool 2.4.0 (new formula)

### PR DESCRIPTION
**Pull request by charleskorn**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by charleskorn** [comment]
I'd also like to add this formula to the [autobump job](https://github.com/Homebrew/homebrew-core/blob/5e2668812f7068dfabb0790e16d8a2f047bc5efb/.github/workflows/autobump.yml). Is that done automatically or should I include that in this PR?

File: Formula/mimirtool.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  livecheck do
+    url :stable
+    regex(/^mimir-(\d+\.\d+\.\d+)$/i)
+  end
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "BUILD_IN_CONTAINER=false", "cmd/mimirtool/mimirtool"
+    bin.install "cmd/mimirtool/mimirtool"
```
Comment (SMillerDev) [inline Formula/mimirtool.rb:17]: Is there no `make install`?

**Comment by SMillerDev** [comment]
> I'd also like to add this formula to the autobump job. Is that done automatically or should I include that in this PR?

You can make a pull request for that after.

File: Formula/mimirtool.rb (hunk 1 above)
Comment (charleskorn) [reply Formula/mimirtool.rb:17]: No - it just has targets to build the various binaries.

File: Formula/mimirtool.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/mimirtool version")
```
[the remark this answers is not in the document]
Comment (charleskorn) [reply Formula/mimirtool.rb:24]: Sorry, I missed that. I've added a better test in 560f8bddf77. (Many of the other commands of `mimirtool` require credentials, so this seems like the best way to test it.)

**Comment by charleskorn** [comment]
@SMillerDev I'm not quite sure why the CI build is failing - it seems to be unrelated to this change (it's complaining about `minio` and `minio-mc`). Am I missing something?

**Comment by chenrui333** [comment]
@charleskorn can you rebase to the latest master branch?

**Comment by charleskorn** [comment]
> @charleskorn can you rebase to the latest master branch?

@chenrui333 done.

**Review by SMillerDev — APPROVED** [review]

**Review by chenrui333 — APPROVED** [review]

**Comment by chenrui333** [comment]
> > @charleskorn can you rebase to the latest master branch?
> 
> @chenrui333 done.

Thank you!

**Comment by chenrui333** [comment]
@charleskorn, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**